tests/python/pants_test/backend/jvm/tasks/test_ivy_updates_integration.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+from textwrap import dedent
+
+from pants.base.build_environment import get_buildroot
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+from pants.util.contextutil import temporary_dir
+
+
+# TODO: re-enable these tests after the ivy-depdendency-update-checker has been released
```

Comment:
nit: s/depdendency/dependency/

Also, you could use @unittest.skip instead of prefixing with xx.